core/com.b2international.snowowl.api.rest/WEB-INF/web.xml

```diff
@@ -1,43 +1,39 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<web-app 
+<web-app
 	xmlns="http://java.sun.com/xml/ns/javaee"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
 	id="WebApp_ID" version="3.0">
 
 	<display-name>com.b2international.snowowl.api.rest</display-name>
 
-	<listener>
-		<listener-class>org.springframework.web.context.ContextLoaderListener</listener-class>
-	</listener>
-
 	<welcome-file-list>
-		<welcome-file>static/index.html</welcome-file>
+		<welcome-file>/static/index.html</welcome-file>
```

Comment:
I can't find any info on whether this absolute path will be "host-relative" or "context-relative". If I point my browser to
`http://localhost:8080/path/to/context`, which is the context path, will this attempt to load `http://localhost:8080/static/index.html` or `http://localhost:8080/path/to/context/static/index.html`?